.travis.yml

```diff
@@ -26,6 +26,8 @@ matrix:
       gemfile: spec/support/Gemfile.rails5
     - rvm: "2.1.10"
       gemfile: spec/support/Gemfile.rails5.1
+    - rvm: "2.2.10"
```

Comment:
A new failure appeared, Rails 5.2 now includes a use of `&.` which fails in < 2.3.